app/routes.py

```diff
@@ -1,2 +1,85 @@
-from flask import Blueprint
+from app import db
+from flask import Blueprint, jsonify, make_response, request
+from app.models.planet import Planet
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+def make_input_valid(number):
+    try:
+        int(number)
+    except:
+        return make_response(f"{number} is not an int!", 400)
+
+def is_parameter_valid(parameter_id):
+    if make_input_valid(parameter_id) is not None:
+        return make_input_valid(parameter_id)
```

Comment:
Python allows to simplify this conditional to check for truthy values:

```suggestion
    if make_input_valid(parameter_id):
        return make_input_valid(parameter_id)
```